README.md

```diff
@@ -26,3 +26,41 @@ This Operator is in the early stages of implementation. For the time being, plea
         *Note*: other provider options can be found in `api/v1alpha1/externaldns_types.go`, e.g. the `ExternalDNSAWSProviderOptions` structure for AWS.
```

Comment:
For now we have just tested on AWS and GCP right ?

Reply:
I tested only on AWS for the moment.